doc/sphinx/source/projects/ad405x_iio/ad405x_iio.rst

```diff
@@ -22,10 +24,10 @@ Introduction
 ============
 
 This page gives an overview of using the ARM platforms supported (default is Mbed) 
```

Comment:
Is default platform mbed in ad405x? We usually have it as stm for newer projects

Reply:
It was for AD4052
The changes in this commit were pushed to bitbucket by Ribhu. Now that AD4050 is released too, it was decided to merge it to Github.